greenplumpython/dataframe.py

```diff
@@ -919,12 +923,24 @@ def save_as(
             f"WITH ({','.join([f'{key}={storage_params[key]}' for key in storage_params.keys()])})"
         )
         df_full_name = f'"{table_name}"' if schema is None else f'"{schema}"."{table_name}"'
+        distribution_clause = (
+            f"""
+                DISTRIBUTED {f"BY ({','.join(distribution_key)})" 
+                if distribution_type == "hash" 
+                else "REPLICATED"
+                if distribution_type == "replicated"
+                else "RANDOMLY"}
```

Comment:
do we have other distribution_type here?

Reply:
We have hash, replicated, randomly and None